app/providers/Claim.tsx

```diff
@@ -23,12 +27,21 @@ export const ClaimProvider: React.FC<{ children: React.ReactNode }> = ({
     setLoading(false);
   }, []);
 
+  const setClaim = async (key: string, value: any) => {
```

Comment:
Does value has a type? string?

Reply:
Because of the flexable nature of the DID schema it can be basically anything from a string, number to full json object with multiple fields.